Sources/VectorExtor/CGPath+DeprecatedExtensions.swift

```diff
@@ -0,0 +1,236 @@
+#if os(macOS) || os(watchOS) || os(iOS) || os(tvOS)
+import CoreGraphics
+
+@available(OSX 10.13, iOS 11.0, tvOS 11.0, watchOS 4.0, *)
+public extension CGPath {
+	@available(*, deprecated, renamed: "segments")
+	enum PathElement: Equatable {
+		case moveTo(point: CGPoint)
+		case addLineTo(point: CGPoint)
+		case addQuadCurveTo(point: CGPoint, controlPoint: CGPoint)
+		case addCurveTo(point: CGPoint, controlPoint1: CGPoint, controlPoint2: CGPoint)
+		case close
+	}
+
+	@available(*, deprecated, renamed: "segments")
+	var elements: [PathElement] {
+		var store = [PathElement]()
+		applyWithBlock { elementsPointer in
+			let element = elementsPointer[0]
+			switch element.type {
+			case .moveToPoint:
+				let point = element.points[0]
+				store.append(.moveTo(point: point))
+			case .addLineToPoint:
+				let point = element.points[0]
+				store.append(.addLineTo(point: point))
+			case .addQuadCurveToPoint:
+				let point = element.points[1]
+				let control = element.points[0]
+				store.append(.addQuadCurveTo(point: point, controlPoint: control))
+			case .addCurveToPoint:
+				let point = element.points[2]
+				let control1 = element.points[0]
+				let control2 = element.points[1]
+				store.append(.addCurveTo(point: point, controlPoint1: control1, controlPoint2: control2))
+			case .closeSubpath:
+				store.append(.close)
+			@unknown default:
+				print("Unknown path element type: \(element.type) \(element.type.rawValue)")
+			}
+		}
+		return store
+	}
+
+	@available(*, deprecated, renamed: "Segment")
+	class PathSection {
+		public let element: PathElement
+		var next: PathSection?
+		//		var previous: PathSection?
```

Comment:
the commented line could be removed now right?

Reply:
fixed